> Please take a look at my suggestions for the tag creation

Regarding the tag name, I noticed you suggested using the `v` prefix. Previously, the `releases/` prefix was used for tags, as can be seen [here](https://github.com/PalladioSimulator/Palladio-Core-PCM/tags). Should we stick to the established format to keep it consistent, or do you explicitly want to switch to the `v` prefix going forward?

I also quickly tested the suggested code in a private repository. Because we are creating an annotated tag with `-a`, Git strictly requires an author. So my test failed with the following error:
```
Committer identity unknown
*** Please tell me who you are.
Run
  git config --global user.email "you@example.com"
  git config --global user.name "Your Name"
fatal: empty ident name (for <runner@runnervmg397c.ajpjuzvd0pcu5m1ughqrvmkcxa.bx.internal.cloudapp.net>) not allowed
Error: Process completed with exit code 128.
```

Do you know if a global Git identity is configured? If not, we could use the GitHub Actions Bot identity (see  [here](https://github.com/actions/checkout/blob/900f2210b1d28bbbd0bd22d17926b9e224e8f231/README.md?plain=1#L327-L328)).

--- a/.github/workflows/build.yml
+++ b/.github/workflows/build.yml
@@ -159,5 +159,7 @@ jobs:
       if: ${{ inputs.release-version != '0.0.0' }}
       shell: bash
       run: |
-        git tag -a v${{ inputs.release-version }} -m "tagging release version ${{ inputs.release-version }}"
-        git push origin v${{ inputs.release-version }}
+        git config user.name "github-actions[bot]"
+        git config user.email "41898282+github-actions[bot]@users.noreply.github.com"
+        git tag -a releases/${{ inputs.release-version }} -m "tagging release version ${{ inputs.release-version }}"
+        git push origin releases/${{ inputs.release-version }}

